.github/workflows/e2e-suite.yml

```diff
@@ -7,16 +7,20 @@ on:
         description: 'Use minimal test account'
         required: false
         default: 'false'
-      module:
-        description: "The module from 'test/integration' to the target to be tested, e.g. 'cli, domains, events, etc'"
+      test_path:
+        description: "The path from 'test/integration' to the target to be tested, e.g. 'cli'"
```

Comment:
We might want to take what is in `dev` for this. I personally think module make things a bit more simpler